lib/msal-angularjs/package-lock.json

```diff
@@ -6184,11 +6184,11 @@
       "dev": true
     },
     "msal": {
-      "version": "0.2.2",
-      "resolved": "https://registry.npmjs.org/msal/-/msal-0.2.2.tgz",
-      "integrity": "sha512-a/3b5HYaJYEebTSI+r+tpIwF9rX6XgTS7mWxaOrMYyg+Szz8SevYd/aHR5c0R63dQkM6/SK3hJ+i1GaGkDVdSA==",
```

Comment:
You should discard these changes to unrelated package lock files.

Reply:
Sure. I auto committed from my vscode, will keep a watch on unrelated changes.